src/integration/java/io/pinecone/helpers/TestResourcesManager.java

```diff
@@ -5,7 +5,7 @@
 import io.pinecone.clients.Pinecone;
 import io.pinecone.exceptions.PineconeException;
 import io.pinecone.proto.DescribeIndexStatsResponse;
-import org.openapitools.control.client.model.*;
+import org.openapitools.db_control.client.model.*;
```

Comment:
example of updating import statement since `db_` is now appended to control and data modules.